integrations/livepeer/README.md

```diff
@@ -0,0 +1,101 @@
+# Livepeer API - React Hook Integration
+
+This integration with the [Livepeer API](https://livepeer.org/) offers a user-friendly approach for managing video content and livestreams directly within the browser, thanks to its suite of reusable React Components.
+
+## Features
+
+- File Uploading Interface: It equips users with an interface to upload MP4 video files to an on-chain server. This feature is critical in preserving data immutability and security.
+
+- Video Playback: It supports video playback capabilities, allowing users to watch previously uploaded videos.
+
+- Livestream Creation: It facilitates the creation of livestreams using OBS (Open Broadcaster Software) or a web browser, offering flexible options to broadcasters.
+
+- Livestream Playback: It enables users to watch livestreams.
+
+- Stream Configuration: It includes a copy-to-clipboard feature that simplifies the use of new live stream configuration information.
+
+- Notifications: It offers toast notifications for copy-to-clipboard actions and error handling, ensuring a smooth user experience.
+
+## API
+
+`useCheckLivepeerApiKey`
+A custom React hook that checks if the provided API Key is valid for use on Livepeer API. It includes the following properties and methods:
+
+- `checkLivepeerApiKey(apiKey)`: A method that tries to delete a hypothetical livestream and according to its errors defines if the provided API Key is valid for Livepeer API.
+  - error with status 404 `NotFoundError` means that the API key is valid and has Full API access
+  - error with status 403 `PermissionError` means that the API key is valid but doesn't have Full API access
+  - error with the message 'Failed to fetch' `FailedToFetchError` means that the API key is not valid, it can be an error to fetch or the valid API Key doesn't have CORS access
+
+`useLivepeerApiKey`
+A custom React hook that provides state regarding Livepeer API Key. It includes the following properties and methods:
+
+- `useLivepeerApiKey`: A method that provides the current state of a custom API Key set by the client 
+- `useIsLivepeerApiKeySet`: A method that checks if the Livepeer API Key is set by the client or the server and returns true if one of the cases is defined and false if neither is.
+
+
+## Components
```

Comment:
Fixing README formatting, otherwise the CI will break

```suggestion
- `useLivepeerApiKey`: A method that provides the current state of a custom API Key set by the client
- `useIsLivepeerApiKeySet`: A method that checks if the Livepeer API Key is set by the client or the server and returns true if one of the cases is defined and false if neither is.

## Components
```